server/db/models/User.js

```diff
@@ -7,6 +7,13 @@ const User = db.define('user', {
     defaultValue: UUIDV4,
     primaryKey: true
   },
+  name:{
```

Comment:
if we do this, we should update the sign up form. If not, we could just make it part of the order.

Reply:
I'll get rid of name, since it doesn't make much of a difference